examples/drupal_loadtest.rs

```diff
@@ -98,26 +98,26 @@ fn drupal_loadtest_front_page(client: &mut GooseClient) {
             client.set_failure();
         },
     }
-}
+} task!(drupal_loadtest_front_page, drupal_loadtest_front_page_task);
```

Comment:
@alsuren It would be ideal if we only had to include the name of the wrapped task here, and could somehow auto-derive/extract the name of the function it's wrapping. Any ideas?

Reply:
this has been resolved with the `task!()` macro.